### PR TITLE
Add AtExceptionStack

### PR DESCRIPTION
**Pull request by sitaram-kalluri**
**- What I did**
1. Add AtExceptionStack class to stack all the exceptions.

### CODE REVIEW

File: at_commons/lib/src/exception/exception_stack.dart [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    for (var element in _exceptionList) {
+      if (element.message == triple.message) {
+        return 0;
+      }
+    }
+    return 1;
+  }
+}
+
+class Triple {
```
Comment (gkc) [inline at_commons/lib/src/exception/exception_stack.dart]: I *really* don't like this class name :-) Can you come up with a name that gives a little more of a clue as to what it is?

**Review by gkc — CHANGES_REQUESTED** [review]

File: at_commons/lib/src/exception/exception_stack.dart (hunk 1 above)
Comment (sitaram-kalluri) [reply at_commons/lib/src/exception/exception_stack.dart]: Renamed class Triple as AtChainedException

File: at_commons/lib/src/exception/at_exception_manager.dart [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import 'package:at_commons/at_commons.dart';
+
+class AtExceptionManager {
+  static AtException createException(AtException atException) {
+    atException.message = atException.getTraceMessage();
+    return atException;
```
Comment (gkc) [inline at_commons/lib/src/exception/at_exception_manager.dart]: What's the expected usage of this? And should it be renamed to 'createExceptionWithFullTraceMessage' or something similar, for clarity?

Comment (sitaram-kalluri) [reply at_commons/lib/src/exception/at_exception_manager.dart]: The createException takes atException and check if its is an instance of AtClientException. If yes, returns as is. Else converts the exception to AtClientException and returns the exception back

**Review by gkc — APPROVED** [review]
LGTM. Will need to test extensively with existing apps of course.

**Pull request merged**